invenio_vocabularies/contrib/subjects/bodc/datastreams.py

```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2024 CERN.
+#
+# Invenio-Vocabularies is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+"""BODC subjects datastreams, readers, transformers, and writers."""
+
+from rdflib.namespace import OWL, RDFS
```

Comment:
minor: this has to be wrapped in a `try/except`, since `rdflib` is an optional dependency now. Check other places in the module where we import `rdflib` to see how we do it